.github/workflows/ci.yml

```
@@ -0,0 +1,198 @@
+          exit 1
+        fi
+      shell: bash
+
+    - name: Run go vet
+      run: go vet ./...
+
+    - name: Run staticcheck
+      run: staticcheck ./...
+
+    - name: Run golangci-lint
+      uses: golangci/golangci-lint-action@v3
```

actionlint (1.7.7)

the runner of "golangci/golangci-lint-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)